README.md

```diff
@@ -17,26 +17,29 @@ The role can be used to configure:
 General
 -------
 
-The role supports two providers: `nm` and `initscripts`. The provider can be configured per host
-via the [`provider`](#provider) variable. In absence of explicit configuration, it is autodetected based on
-the distribution. The `nm` provider is used by default on RHEL7 and
-`initscripts` on RHEL6. However, note that the provider is not tied to a certain distribution,
-given that the required API is available. For `nm` this means that at least version 1.2 of NetworkManager's
-API is available. For `initscripts`, it requires the legacy network service as commonly available
-on Fedora/RHEL.
-
-For each host a list of networking profiles can be configure via the `network` variable.
+The role supports two providers: `nm` and `initscripts`. The provider can be
+configured per host via the [`network_provider`](#provider) variable. In
+absence of explicit configuration, it is autodetected based on the
+distribution. The `nm` provider is used by default on RHEL7 and `initscripts`
+on RHEL6. However, note that the provider is not tied to a certain
+distribution, given that the required API is available. For `nm` this means
+that at least version 1.2 of NetworkManager's API is available. For
+`initscripts`, it requires the legacy network service as commonly available on
+Fedora/RHEL.
+
+For each host a list of networking profiles can be configure via the
+`network_connections` variable.
 
 - For initscripts, profiles correspond to ifcfg files in `/etc/sysconfig/network-scripts/ifcfg-*`.
 
 - For NetworkManager, profiles correspond to connection profiles as handled by NetworkManager.  Fedora and RHEL use the `rh-plugin` for NetworkManager which also writes configuration files to `/etc/sysconfig/network-scripts/ifcfg-*` for compatibility.
 
 Note that the role primarily operates on networking profiles (connections) and
-not on devices. For example, in the role you would not configure the current IP address
-of an interface. Instead, you create a profile with a certain IP configuration and
-optionally activate the profile on a device. Which means, to apply the configuration
-to the actual networking interface.
-
+not on devices but it defaults to use the profile name as the interface name.
+But it is also possible to create generic profiles, by creating for example a
+profile with a certain IP configuration and optionally activate the profile on
+a device. Which means, to apply the configuration to the actual networking
+interface.
```

Comment:
How does one do that?

Reply:
This needs to be done separately, e.g. by using `nmcli`.

Reply:
Thank you for clarifying this, because from the previous version of the description it was not obvious at all that this would need to be done outside of the role.